.github/actions/composite/setupNode/action.yml

```diff
@@ -4,10 +4,6 @@ description: Set up Node
 runs:
   using: composite
   steps:
-    - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
```

Comment:
Mentioned on a commit, but putting here:

I don't think we can remove this line, without it we do not have the nvm version in .nvmrc to use below. I think we've seen failures because of this.

cc @luacmartins @mountiny

Can we just change the workflow that was using this composite step instead of changing this instance?

Reply:
cc @staszekscp

Reply:
I've made sure the `checkout` action is run always where `setupNode` appears. If you prefer I may revert this change in the next PR. I just though we'll have to use different refs and the `checkout` action is not strictly connected to the node setup so it would be better to extract it.

Reply:
Got it, that's fair!

Reply:
We can keep as is 👍